recipes/tree-gen/all/test_package/src/example.cpp

```diff
@@ -0,0 +1,290 @@
+#include "output/directory.actual.hpp"  // the generated file
+#include "utils.hpp"
+#include "version.hpp"  // tree-gen version
+
+#include <cstdio>
+#include <iostream>
+#include <sstream>  // ostringstream
+#include <stdexcept>
+
+
+void print_tree_gen_version() {
+    std::printf("tree-gen version: %s\n", TREE_GEN_VERSION);
+    std::printf("tree-gen release year: %s\n", TREE_GEN_RELEASE_YEAR);
+}
+
+
+// Note: the // comment contents of main(), together with the MARKER lines and the output of the program,
+// are used to automatically turn this into a restructured-text page for ReadTheDocs.
+
+int main() {
+    print_tree_gen_version();
+
+    // **********************
+    // Directory tree example
+    // **********************
```

Comment:
I think we can skip this for now. We are thinking of having an extra "test" package (Something like example_package) that may contain example usages for better UX, but test packages should only test that the library was packaged correctly, so we can skip most of this)

Reply:
Fair enough. I've removed most of the code in the `main` function.